From 242fd2cbad1075d4cc0e3a3b64652dbc766bd117 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Sun, 21 Aug 2016 23:20:11 +0200
Subject: [PATCH 10/10] lit/setup.py: Install as llvm-lit (as cmake expects it)

Modify the setup.py for lit to install the entry point as llvm-lit
(instead of lit) since this is the name expected by functions in
AddLLVM.cmake.

Patch: https://reviews.llvm.org/D23743
---
 utils/lit/setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/lit/setup.py b/utils/lit/setup.py
index 10de6bb..c57e0aa 100644
--- a/utils/lit/setup.py
+++ b/utils/lit/setup.py
@@ -71,7 +71,7 @@ http://llvm.org/svn/llvm-project/llvm/trunk/utils/lit.
     packages = find_packages(),
     entry_points = {
         'console_scripts': [
-            'lit = lit:main',
+            'llvm-lit = lit:main',
             ],
         }
 )
-- 
2.9.3

